src/reducers/routerReducer.js

```diff
@@ -15,26 +15,10 @@ const INITIAL_STATE = {
           name: 'Mission and Values',
           linkTo: '/about/mission-and-values'
         },
-        {
-          name: 'History',
-          linkTo: '/about/history'
-        },
-        {
-          name: 'HO Center',
-          linkTo: '/about/hawaii-okinawan-center'
-        },
```

Comment:
lets not delete these just yet

could you change the name above to "Mission and Purpose" and do the same with the linkTo